subtrack.MAUI/Pages/Monthly.razor

```diff
@@ -0,0 +1,50 @@
+﻿@page "/monthly"
+@using System.Globalization;
+@using subtrack.MAUI.Services.Abstractions;
+
+@inject subtrack.DAL.SubtrackDbContext Db
+
+<div class="container text-center">
+    <div class="row">
+        <div class="col">
+            <h6 class="pt-1">Monthly</h6>
+        </div>
+    </div>
+</div>
+
+<div class="accordion accordion-flush" id="accordionFlush">
+
+    @foreach (var month in _months.Select((Value, Index) => new { Value, Index }))
+    {
+        <div class="accordion-item">
+            <h2 class="accordion-header" id="@($"flush-heading{month.Index}")">
+                <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="@($"#flush-collapse{month.Index}")" aria-expanded="false" aria-controls="@($"flush-collapse{month.Index}")">
+                    <div class="d-flex justify-content-between">
+                        <span>@month.Value | @($"{_subscriptionsCost:C}")</span>
+                    </div>
+                </button>
+            </h2>
+            <div id="@($"flush-collapse{month.Index}")" class="accordion-collapse collapse" aria-labelledby="@($"flush-heading{month.Index}")" data-bs-parent="#accordionFlush">
+                <div class="container mt-3">
+                    @foreach (var sub in Db.Subscriptions)
+                    {
+                        <p>@sub.Name $@sub.Cost</p>
+                    }
+                </div>
+            </div>
+        </div>
+    }
+</div>
+
+
+@code{
+    decimal _subscriptionsCost = 0;
+    IEnumerable<Subscription> _subscriptions = new List<Subscription>();
+    string[] _months = DateTimeFormatInfo.CurrentInfo.MonthNames;
```

Comment:
add a variable for specifying the start month since we only want to display this month and the ones after it

Reply:
So in case we're in December we'll only show that one month? We're not going over to the next year right?

Ideally I'd say maybe loop around and always show 12 months total. Like if we're in June, we show June - December 2023 and January - May 2024

Reply:
Second option is correct. Show this month and all months for a total of 12 months. Just make sure you start this month since the ones before are not relevant

Reply:
The months that are not in this year should have the suffix year added to them
f.e a month this year's title would be: January
If its january next year: January 2024